src/components/VerschillenGebied.vue

```diff
@@ -285,23 +392,51 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@import "../../static/ams.scss";
+@import "~stijl/src/styles/generic/ams-colorpalette.scss";
 
-.map {
-  height: 350px;
-  margin-top: 5px;
-  margin-bottom: 5px;
-}
+  .map {
+    height: 400px;
+  }
 
-.select {
-  width: 100%;
-}
+  table {
```

Comment:
does this work in scoped style? I guess it is application wide

Reply:
removed it, as there is no table in this component anymore